documentation/tut/toy-example-process.rst

```diff
@@ -0,0 +1,110 @@
+.. _tut_toy_schedule_process:
+
+
+
+Toy example III: Computing schedules for processes
+====================================================================
+
+Until this point we've been using a static battery, one of the most flexible energy assets, to reduce electricity bills. 
```

Comment:
I'd add: "A battery can modulate rather freely, and both charge and discharge."

(this feeds better into the next sentence)

Reply:
Has my suggestion been rejected?

If so, please say so. Better yet of course, give a reason :)

Reply:
Thanks for re-reviewing the page!

I thought I actually addressed both requests, but apparently they got deleted on a branch switch (maybe I kept it in a stash...).